### PR TITLE
Open the bus once in networkctl

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

File: src/basic/macro.h [hunk 1]
```diff
@@ -254,6 +254,14 @@ static inline int __coverity_check_and_return__(int condition) {
 #define assert(expr) assert_message_se(expr, #expr)
 #endif
 
+/* This passes the argument through after (if asserts are enabled) checking that it is not null. */
+#define ASSERT_PTR(expr)                        \
```
Comment (medhefgo) [inline src/basic/macro.h]: Maybe put that in macro-fundamental. It could replace assert_alloc.

**Review by yuwata — APPROVED** [review]
LGTM.

File: src/basic/macro.h (hunk 1 above)
Comment (yuwata) [reply src/basic/macro.h]: Moved.

**Comment by yuwata** [comment]
The macro is moved to fundamental. Upgrading the green label.

**Comment by keszybz** [comment]
Thanks.

**Pull request merged**